botocore/utils.py

```diff
@@ -48,6 +48,20 @@
 RETRYABLE_HTTP_ERRORS = (requests.Timeout, requests.ConnectionError)
 S3_ACCELERATE_WHITELIST = ['dualstack']
 
+CONTROLS = {
+    '\n': '\\n',
+    '\r': '\\r',
+    '\t': '\\t',
+}
+# Combines all CONTROLS keys into a big or regular expression
+ESCAPE_CONTROLS_RE = re.compile('|'.join(map(re.escape, CONTROLS)))
```

Comment:
Yeah that's how I was planning on doing it initially but ended up following the regex example above it. I'll go ahead and mark it as private when I move it to `docs/utils.py`